compiler/packages/babel-plugin-react-compiler/src/CompilerError.ts

```diff
@@ -260,3 +426,32 @@ export class CompilerError extends Error {
     });
   }
 }
+
+function printErrorSummary(severity: ErrorSeverity, message: string): string {
```

Comment:
Nice, this is a lot more understandable for people reading lint errors than `InvalidJS: ...`